templates/csharp/custom-copilot-assistant-assistants-api/infra/azure.bicep.tpl

```diff
@@ -97,7 +97,7 @@ resource webApp 'Microsoft.Web/sites@2021-02-01' = {
           value: azureOpenAIDeploymentName
         }
         {
-          name: 'Azure__OpenAIDeploymentName'
+          name: 'Azure__OpenAIAssistantId'
```

Comment:
__ or _ ?

Reply:
double underscore because it is a nested structure.
![image](https://github.com/user-attachments/assets/b7974805-4d4b-480a-bcd2-8745ab19e022)